apps/api/openapi/openapi.json

```diff
@@ -14,13 +14,17 @@
     }
   },
   "servers": [
+    {
```

Comment:
I had to make some fixes to the spec. Since the spec now defines the SDK, it **must** match the actual api behavior.